_includes/configuration/powerpaste_clean_filtered_inline_elements.md

```diff
@@ -0,0 +1,9 @@
+### `powerpaste_allow_local_images`
```

Comment:
This whole file has been copied incorrectly it seems, as this is the content for `powerpaste_allow_local_images` when it should be `powerpaste_clean_filtered_inline_elements`.

- [ ] Fix incorrect setting content

Reply:
fixed in [`daef718` (#2038)](https://github.com/tinymce/tinymce-docs/pull/2038/commits/daef718a0fde627f3385621857d5eb387543516b)